main/integration-tests/support/resources/studies/study.js

```diff
@@ -42,8 +42,8 @@ class Study extends Resource {
     return new StudyFiles({ clientSession: this.clientSession, parent: this });
   }
 
-  // StudyFiles is a child resource operations helper
-  uploadRequests() {
+  // StudyUploadRequests is a child resource operations helper
+  uploadRequest() {
```

Comment:
Since we are calling it 'uploadRequest'.  Should you also rename the StudyUploadRequests to just StudyUploadRequest?  same for study-upload-requests.js

Reply:
Keeping the class and file named as-is for now, since its API route matches that nomenclature